public class org.bouncycastle.jce.exception.ExtCertPathValidatorException extends java.security.cert.CertPathValidatorException implements org.bouncycastle.jce.exception.ExtException {
  private java.lang.Throwable cause;

  public org.bouncycastle.jce.exception.ExtCertPathValidatorException(java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: invokespecial #19                 // Method java/security/cert/CertPathValidatorException."<init>":(Ljava/lang/String;)V
       5: return

  public org.bouncycastle.jce.exception.ExtCertPathValidatorException(java.lang.String, java.lang.Throwable);
    Code:
       0: aload_0
       1: aload_1
       2: invokespecial #19                 // Method java/security/cert/CertPathValidatorException."<init>":(Ljava/lang/String;)V
       5: aload_0
       6: aload_2
       7: putfield      #18                 // Field cause:Ljava/lang/Throwable;
      10: return

  public org.bouncycastle.jce.exception.ExtCertPathValidatorException(java.lang.String, java.lang.Throwable, java.security.cert.CertPath, int);
    Code:
       0: aload_0
       1: aload_1
       2: aload_2
       3: aload_3
       4: iload         4
       6: invokespecial #20                 // Method java/security/cert/CertPathValidatorException."<init>":(Ljava/lang/String;Ljava/lang/Throwable;Ljava/security/cert/CertPath;I)V
       9: aload_0
      10: aload_2
      11: putfield      #18                 // Field cause:Ljava/lang/Throwable;
      14: return

  public java.lang.Throwable getCause();
    Code:
       0: aload_0
       1: getfield      #18                 // Field cause:Ljava/lang/Throwable;
       4: areturn
}
